app/controllers/repositories_controller.rb

```diff
@@ -0,0 +1,74 @@
+class RepositoriesController < ApplicationController
+  after_action :mark_everything_as_read, only: [:index, :show]
+  before_action :print_read_status, only: :index
+
+  @client = Octokit::Client.new(:access_token => ENV['PERSONAL_ACCESS_TOKEN'])
+
+  def index
+    # update_repo_tags
+    @repositories = Repository.all.sort_by(&:latest_release_date).reverse!
+    new
+    @repositories
+  end
+
+  def show
+    @repository = Repository.find(params[:id])
+    Services::Monitoring.set_repo_latest_release(@repository, @client)
+    @repository.save
+  end
+
+  def new
+    @repository = Repository.new
+  end
+
+  def create
+    puts "hello!"
+    @repository = Repository.new(repository_params)
+
+    if @repository.save
+      update_repo_tags
+      redirect_to repositories_path
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @repository = Repository.find(params[:id])
+    @repository.destroy
+
+    redirect_to root_path, status: :see_other
```

Comment:
Is See Other normal for a delete request? (I'm unsure - it looks strange to me but it's been a while since I've tried to think about the correct status codes to use.)

Reply:
I most certainly wouldn't know if it's normal per se but I copied this from [railsguides dot org](https://guides.rubyonrails.org/getting_started.html#deleting-an-article) so I suppose it should be legit?